doc/ch-image.rst

```diff
@@ -98,13 +101,15 @@ Common options placed before or after the sub-command:
     subprocesses. Profile data should still be written on fatal errors, but
     not if the program crashes.
 
+  :code:`-q, --quiet`
+    Be quieter; can be repeated. Incompatible :code:`-v` and suppresses
+    :code:`--debug` regardless of option order. See the :ref:`FAQ entry on
```

Comment:
I wonder if we should revisit the `--debug` incompatibility? It seems orthogonal now. 

Also `--debug` seems to be undocumented here.